zkg.meta

```diff
@@ -11,6 +11,7 @@ test_command  = unset -v CXX CXXFLAGS LD LDFLAGS && cd tests && PATH=$(zkg confi
 # express that dependency due to zeek/package-manager#106
 # depends = spicy-plugin >=1.2.2
 depends = http://github.com/zeek/spicy-ldap ==0.0.2
+    http://github.com/zeek/spicy-pe ==0.0.1
```

Comment:
Question: Do we really want to pin versions? I get why that's desirable from our perspective, but I'm wondering if that's maintainable: with so many external analyzers, it'll be regularly out of date.

Reply:
I opened #112 for moving to unpinned deps and will update this PR once that one is merged.